### PR TITLE
Fix errors across all docs

### PR DESCRIPTION
**Pull request by WiXSL**

### CODE REVIEW

File: docs/ArrayField.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Display a collection using `<Field>` child components.
 
 Ideal for embedded arrays of objects, e.g. `tags` and `backlinks` in the following `post` object:
 
-```js
+```json
```
Comment (fzaninotto) [inline docs/ArrayField.md:12]: no, the snippet isn't valid JSON

File: docs/Fields.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ export const BookShow = () => (
 
 When rendered for the following record:
 
-```jsx
+```json
```
Comment (fzaninotto) [inline docs/Fields.md:67]: same for your 3 editions in this file

File: docs/FileField.md [hunk 3]
```diff
@@ -59,13 +59,13 @@ The optional `title` prop points to the file title property, used for `title` at
 
 If the record actually contains an array of files in its property defined by the `source` prop, the `src` prop will be needed to determine the `href` value of the links, for example:
 
-```js
+```json
```
Comment (fzaninotto) [inline docs/FileField.md:62]: same

File: docs/ImageField.md [hunk 4]
```diff
@@ -56,13 +56,13 @@ The optional `title` prop points to the picture title property, used for both `a
 
 If the record actually contains an array of images in the property defined by the `source` prop, the `src` prop will be needed to determine the `src` value of the images, for example:
 
-```js
+```json
```
Comment (fzaninotto) [inline docs/ImageField.md:59]: same

File: docs/Inputs.md [hunk 5]
```diff
@@ -266,17 +266,17 @@ For instance, let's write a component to edit the latitude and longitude of the
 import { useController } from 'react-hook-form';
 
 const LatLngInput = () => {
-    const input1 = useController({ name: 'lat' });
```
Comment (fzaninotto) [inline docs/Inputs.md:269]: I really preferred the previous version

File: docs/TranslatableFields.md [hunk 6]
```diff
@@ -7,7 +7,7 @@ title: "The TranslatableFields Component"
 
 You may have fields which are translated in multiple languages and want users to verify each translation. To display them, you can use the `<TranslatableFields>` component, which expects the translatable values to have the following structure:
 
-```js
+```json
```
Comment (fzaninotto) [inline docs/TranslatableFields.md:10]: same

File: docs/TranslatableInputs.md [hunk 7]
```diff
@@ -11,7 +11,7 @@ You may have inputs which are translated in multiple languages and want users to
 
 It expects the translatable values to have the following structure:
 
-```js
```
Comment (fzaninotto) [inline docs/TranslatableInputs.md:14]: same

File: docs/Translation.md [hunk 8]
```diff
@@ -352,7 +352,7 @@ Beware that users from all around the world may use your application, so make su
 
 The `message` returned by the `polyglotI18nProvider` function argument should be a dictionary where the keys identify interface components, and values are the translated string. This dictionary is a simple JavaScript object looking like the following:
 
-```js
+```json
```
Comment (fzaninotto) [inline docs/Translation.md:355]: same

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: docs/Inputs.md (hunk 5 above)
Comment (WiXSL) [reply docs/Inputs.md:269]: For the `LatLngInput ` component, I change a syntax error, what do you mean by the previous version?

Comment (fzaninotto) [reply docs/Inputs.md:269]: these 2 lines, you've just removed blank spaces and that makes the code harder to read

**Comment by fzaninotto** [comment]
Your PR still turns some JS blocks into JSON blocks  and it shouldn't

File: docs/Inputs.md (hunk 5 above)
Comment (WiXSL) [reply docs/Inputs.md:269]: Sorry you're right

**Review by fzaninotto — APPROVED** [review]

**Pull request merged**